### DISCUSSION TITLE
Can I get some help with a hacked account?

### DISCUSSION THREAD

**Discussion by GABO1423** [community/community#138846] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Back in late July, my GitHub account was hacked while I was sleeping. This is an account I use for university work and have been unable to recover it due to very slow response times from GitHub. That ticket has been in limbo and I resume my studies soon, any chance someone at GitHub can look at this?

Thanks in advance.

**Comment by ebndev** [comment]
Hi @gabo14231,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
